ratchet_core/src/split/mod.rs

```diff
@@ -255,7 +262,15 @@ where
 
     /// Returns whether this WebSocket is closed.
     pub fn is_closed(&self) -> bool {
-        self.closed.load(Ordering::Relaxed)
+        self.close_state.load(Ordering::Relaxed) == STATE_CLOSED
```

Comment:
I think this should maybe be sequentially consistent as you don't know what context this will be used in (and why the caller might want to know).